cache/cache.go

```diff
@@ -239,29 +266,29 @@ func (s *redisStore) getAndRefresh(ctx context.Context, record string, expiratio
 // SetAuthRecord sets a pair, granted option and expiration time.
 func (s *goStore) SetAuthRecord(ctx context.Context, username, password string, granted string) error {
 	record := toAuthRecord(username, password, s.h)
-	s.client.Set(record, granted, s.authExpiration)
+	s.client.Set(record, granted, expirationWithJitter(s.authExpiration, s.authJitter))
 
 	return nil
 }
 
 //SetAclCache sets a mix, granted option and expiration time.
 func (s *goStore) SetACLRecord(ctx context.Context, username, topic, clientid string, acc int, granted string) error {
 	record := toACLRecord(username, topic, clientid, acc, s.h)
-	s.client.Set(record, granted, s.authExpiration)
+	s.client.Set(record, granted, expirationWithJitter(s.aclExpiration, s.aclJitter))
 
 	return nil
 }
 
 // SetAuthRecord sets a pair, granted option and expiration time.
 func (s *redisStore) SetAuthRecord(ctx context.Context, username, password string, granted string) error {
 	record := toAuthRecord(username, password, s.h)
-	return s.setRecord(ctx, record, granted, s.authExpiration)
+	return s.setRecord(ctx, record, granted, expirationWithJitter(s.authExpiration, s.authJitter))
 }
 
 //SetAclCache sets a mix, granted option and expiration time.
 func (s *redisStore) SetACLRecord(ctx context.Context, username, topic, clientid string, acc int, granted string) error {
 	record := toACLRecord(username, topic, clientid, acc, s.h)
-	return s.setRecord(ctx, record, granted, s.authExpiration)
+	return s.setRecord(ctx, record, granted, expirationWithJitter(s.aclExpiration, s.aclJitter))
```

Comment:
Again, nice catch! It does seem like you brought up these wrong expirations some time ago though, didn't you? If so, pulling `master` and rebasing would be needed.

Reply:
hum... yes I did in PR #122 which is merged in master. But because this PR start from PR #122, I would have expected github to hide this diff once the first got merged.